app/src/main/res/layout/fragment_directions_form.xml

```diff
@@ -150,7 +147,6 @@
 		android:paddingEnd="4dp"
 		android:paddingRight="4dp"
 		android:gravity="center"
-		android:textColor="@color/md_grey_300"
```

Comment:
I don't think it's a good idea to have text color set by Material You.

Reply:
We dont? We use the default text color here, which is a dark/light grey depending on the current device theme.